edm4eic.yaml

```
@@ -526,6 +526,22 @@ datatypes:
     OneToManyRelations:
       - edm4eic::ReconstructedParticle hadrons // Reconstructed hadrons used in calculation
 
+  ## ==========================================================================
+  ## Jets
+  ## ==========================================================================
+
+  edm4eic::Jet:
+    Description:  "A reconstructed jet, inspired by the FastJet PseudoJet"
+    Author: "D. Anderson"
+    Members:
+      - float             area             // jet area [sr]
```

```suggestion
      - float             area             // jet area
```
$dA = d\eta d\phi$ , where`\eta` is not in radians so $dA$ is not a solid angle